CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- Convert non-RGB images to RGB (always) in the utilities. (1394)
```

Comment:
Let's phrase this as a bug fix: "Fixed a bug where greyscale images were not properly converted to RGB when loaded" or something like that

Reply:
Sure, @ethanwharris - I'll do this once the CI is green. Thanks!